manifests/site.pp

```diff
@@ -2,7 +2,7 @@
   case $::operatingsystem {
     'Ubuntu': {
       case $::lsbdistcodename {
-        'bionic': {
+        'focal': {
```

Comment:
This won't work until https://github.com/aad-for-linux/aad-for-linux/issues/8 is implemented.